go.mod

```diff
@@ -4,6 +4,7 @@ go 1.14
 
 require (
 	cloud.google.com/go v0.44.3
+	git.blindage.org/21h/hcloud-dns v0.0.0-20200507195107-b8e668eab385
```

Comment:
@21h What do you think to release some version in your library. Than it would easy to track changes between released during the upgrades?